session_rpcserver.go

```diff
@@ -353,16 +350,6 @@ func (s *sessionRpcServer) resumeSession(ctx context.Context,
 	pubKey := sess.LocalPublicKey
 	pubKeyBytes := pubKey.SerializeCompressed()
 
-	// We only start non-revoked, non-expired LiT sessions. Everything else
-	// we just skip.
-	if sess.State != session.StateInUse &&
-		sess.State != session.StateCreated {
-
-		log.Debugf("Not resuming session %x with state %d", pubKeyBytes,
-			sess.State)
-		return nil
-	}
```

Comment:
Is it safe to assume that the passed in session will not have an revoked/expired state? I see that all current calls to `resumeSession` ensure that the sesion is in a valid state, but I'm thinking about future code that may not have these safeguards in place.

If you think it's best to still remove this, then the function comment should be updated.

Reply:
cool yeah currently all the session passed to this method will already be in the Created/InUse state since it is either called from `start` which now only fetches sessions in these states or it is called on session creation which will defs have the CreatedState. 

GOod call - will update the comment 👍